packages/web-forms/src/components/FormHeader.vue

```diff
@@ -124,22 +150,113 @@ const handleLanguageChange = (event: FormLanguage) => {
 	}
 }
 
-.smaller-screens {
-	background-color: var(--surface-0);
-	filter: drop-shadow(0px 2px 6px rgba(0, 0, 0, 0.15)) drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.30)) ;
+.form-error-message.p-message.p-message-error {
+	border-radius: 10px;
+	background-color: var(--error-bg-color);
+	border: 1px solid var(--error-text-color);
+	width: 70%;
+	margin: 0rem auto 1rem auto;
+	position: sticky;
+	top: 0;
+	// Some PrimeVue components use z-index.
+	// Default value for those are either 1000 or 1100
+	// So 5000 here is safe.
+	z-index: 5000;
+	display: none;
 
-	h1 {
-		padding-left: 10px;
-		font-size: 1.5rem;
+	:deep(.p-message-wrapper) {
+		padding: 0.25rem 0.75rem;
+		flex-grow: 1;
 	}
 
-	.form-options{
-		padding-right: 10px;
+	:deep(.p-message-text){
+		font-weight: 400;
+		display: flex;
+		width: 100%;
+		align-items: center;
+
+		div {
+			flex-grow: 1;
+		}
+
+		.fix-errors {
+			color: var(--error-text-color);
+			font-weight: 400;
+
+			&:hover, &:focus {
+				background: #F9DEDC;
+			}
+
+			&:active {
+				background: var(--red-100);
+			}
+		}
 	}
-	
-	.btn-menu{
-		color: var(--surface-900);
+}
+
+.smaller-screens {
+	.title-bar{
+		background-color: var(--surface-0);
+		filter: drop-shadow(0px 2px 6px rgba(0, 0, 0, 0.15)) drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.30)) ;
+
+		h1 {
+			padding-left: 10px;
+			font-size: 1.25rem;
+			font-weight: 400;
+		}
+		
+		.form-options {
+			padding-right: 10px;
+			flex-grow: 1;
+			min-width: 50px;
+			container-type: size;
+			container-name: formOptionsContainer;
+			height: 40px;
+			margin-top: 6px;
+
+			.multilingual{
+				display: flex;
+				justify-content: end;
+				gap: 0.5rem;
+
+				.btn-menu{
+					color: var(--surface-900);
+				}
+				.print-button {
+					display: none;
+				}
+				.language-changer {
+					display: none;
+				}
+			}
+
+			@container formOptionsContainer (min-width: 240px) {
```

Comment:
Hey look at that, it's a container query! Cool to see it in action. We may want to up this `min-width` (and the `.language-changer` `max-width`) some, but otherwise 🎉